apps/core/src/dashboards/entities/dashboard-widget.entity.ts

```diff
@@ -1,23 +1 @@
-import { IsEnum, IsString, Length } from 'class-validator';
```

Comment:
This validation was incorrect because it does not reflect the actual dashboard definition api. Will need to revisit this soon.